android/settings.gradle

```diff
@@ -1,4 +1,8 @@
 rootProject.name = 'LunesAppUI'
+include ':react-native-tts'
+project(':react-native-tts').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-tts/android')
+include ':react-native-tts'
+project(':react-native-tts').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-tts/android')
```

Comment:
Is this supposed to be here twice?

Reply:
I don't really know! I didn't add anything here, they were added when I installed the library! @DaveOuds